app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    _, file, new_file = command.split()
+
+    if file != new_file:
```

Comment:
Read the checklist 


![image](https://user-images.githubusercontent.com/104218550/220341887-5e9ea686-48c1-4c5b-bf82-8efb2ba3d20e.png)